scripts/rustodon.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Rustodon ActivityPub compatible micro-blogging
+After=network.target
+ConditionPathExists=/srv/rustodon/.env
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/default/rustodon
+WorkingDirectory=/srv/rustodon
+ExecStart=/srv/rustodon/target/release/rustodon -- $DAEMON_OPTS
+#Restart=on-failure
```

Comment:
Is there a reason we don't restart on failure?

Reply:
You're right. No reason to not restart.